Makefile

```diff
@@ -166,60 +228,106 @@ build-oss: build
 #  2. timescale/timescaledb-ha:pg12-ts1.7-latest
 #  3. timescale/timescaledb-ha:pg12.3-ts1.7-latest
 #  4. timescale/timescaledb-ha:pg12.3-ts1.7.1-latest
-#  5. timescale/timescaledb-ha:pg12.3-ts1.7.1-pN
-#
-# Tag 5 is immutable, and for every time we publish that image, we increase N by 1,
-# we start with N=0
-publish: publish-mutable publish-next-patch-version
 
-is_ci:
-	@if [ "$${CI}" != "true" ]; then echo "environment variable CI is not set to \"true\", are you running this in Github Actions?"; exit 1; fi
+.PHONY: build-oss
+build-oss: # build an OSS-only image
+build-oss: OSS_ONLY=true
+build-oss: DOCKER_TAG_POSTFIX=-oss
+build-oss: release
+
+.PHONY: publish-combined-builder-manifest
+publish-combined-builder-manifest: # publish a combined builder image manifest
+	@echo "Creating manifest $(DOCKER_BUILDER_URL) that includes $(DOCKER_BUILDER_URL)-amd64"
+	amddigest_image="$$(./fetch_tag_digest $(DOCKER_BUILDER_URL)-amd64)"
+	echo "AMD: $$amddigest_image"
```

Comment:
Arm info is missing. is this the case to add it?

Reply:
It is. Arm was working 100%, but we need to sort out a secure arm64 builder infrastructure. I've got a PoC for that, but I need to sort out the webhooks for it from GH before re-introducing arm64.